scribbl/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :exception
```

Comment:
you can use `authenticate_user` here to restrict creation/editing/deletion to logged in users